app/src/pages/inside/launchesPage/LaunchToolbar/actionPanel/actionPanel.jsx

```diff
@@ -7,9 +8,13 @@ import styles from './actionPanel.scss';
 
 const cx = classNames.bind(styles);
 
-export const ActionPanel = () => (
+export const ActionPanel = ({ hasErrors }) => (
   <div className={cx('action-panel')}>
-    <div className={cx('breadcrumb')} />
+    {hasErrors ? (
+      <GhostButton>Proceed Valid Items</GhostButton>
+    ) : (
+      <div className={cx('breadcrumb')} />
```

Comment:
As I see on dev server, Breadcrumbs visibility do not depends from error status of selected launches. If at least 1 item is selected - breadcrumbs block becomes hidden.